spec/support/pages/accounts_edit_page.rb

```diff
@@ -19,10 +19,12 @@ def add_account(account_name)
   end
 
   set_url '/v3#/accounts/edit'
-  set_url_matcher /\/v3#\/accounts\/edit$/
+  set_url_matcher %r{/v3#/accounts/edit$}
 
-  sections :asset_categories,     CategorySection, ".m-accountType:nth-child(1) .m-category"
-  sections :liability_categories, CategorySection, ".m-accountType:nth-child(2) .m-category"
+  sections :asset_categories,     CategorySection,
+    '.m-accountType:nth-child(1) .m-category'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.